src/main/scala/io/iohk/ethereum/network/package.scala

```diff
@@ -19,4 +25,38 @@ package object network {
       Hex.decode(nodeId)
     curve.getCurve.decodePoint(bytes)
   }
+
```

Comment:
This probably deserves it's own object in ti's own file.And I think it's orthogonal to network. Possibly 'crypto' or 'utils'